src/Dapr.AI/Conversation/Models/Response/DaprConversationResponse.cs

```diff
@@ -0,0 +1,26 @@
+﻿// ------------------------------------------------------------------------
+// Copyright 2024 The Dapr Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------------------------------------------
+
+namespace Dapr.AI.Conversation.Models.Response;
+
+/// <summary>
+/// The response for a conversation.
+/// </summary>
+/// <param name="Outputs">The collection of conversation results.</param>
+public record DaprConversationResponse(List<DaprConversationResult> Outputs)
```

Comment:
Any reason why this `record` uses *both* constructor and property initialization rather than one or the other?  (Others in this PR use only constructor initialization.  I don't have a huge opinion either way, so long as it's consistent across the API.)

Reply:
Not a good one - I originally wrote the implementation based on the values in the proposal, but then had to run back through them all to add everything not in the proposal, but added to the implementation, so this is likely an artifact of that. Pushed this into the constructor.